policy/modules/services/console_setup.fc

```diff
@@ -0,0 +1,14 @@
+/etc/console-setup	-d	gen_context(system_u:object_r:console_setup_conf_t,s0)
+/etc/console-setup/.*	gen_context(system_u:object_r:console_setup_conf_t,s0)
+/etc/console-setup/cached_setup_font\.sh	--	gen_context(system_u:object_r:console_setup_exec_t,s0)
+/etc/console-setup/cached_setup_terminal\.sh	--	gen_context(system_u:object_r:console_setup_exec_t,s0)
+/etc/console-setup/cached_setup_keyboard\.sh	--	gen_context(system_u:object_r:console_setup_exec_t,s0)
```

Comment:
Subjective but i would probably have omited these and instead i would have allowed consolesetup_t to execute console_setup_conf_t files instead of console_setup_exec_t files

`can_exec(console_setup_t, console_setup_conf_t)`